File: backend/flight_booking_project/src/main/java/com/javaweb/flight_booking_project/dto/response/SeatResponse.java
package com.javaweb.flight_booking_project.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;

@Data
@Getter
@Setter
public class SeatResponse {
    private Long id;
    private String seatNumber;
    private String seatClass;
    private boolean available;
    private double price;
}

File: mobile/lib/data/model/seat_response.dart
class SeatResponse {
  int id;
  String seatNumber;
  String seatClass;
  double price;

  SeatResponse({
    required this.id,
    required this.seatNumber,
    required this.seatClass,
    required this.price,
  });

  factory SeatResponse.fromJson(Map<String, dynamic> json) {
    return SeatResponse(
      id: json['id'],
      seatNumber: json['seatNumber'],
      seatClass: json['seatClass'],
      price: json['price'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'seatNumber': seatNumber,
      'seatClass': seatClass,
      'price': price,
    };
  }
}